src/engine/strat_engine/blockdevmgr.rs

```diff
@@ -298,7 +343,9 @@ pub fn initialize(pool_uuid: &PoolUuid,
         } else {
             // TODO: check the return values and update state machine on failure
             let _ = BDA::wipe(&mut f);
-            let _ = wipe_blockdevs(bds);
+            let _ = wipe_blockdevs(bds.into_iter()
```

Comment:
I don't think ```wipe_blockdevs()``` needs a ```Vec``` of reference counted BlockDevs.

Reply:
Oh, just a `&[Rc<RefCell<BlockDev>>]`? Ok.